circle.yml

```diff
@@ -19,12 +19,10 @@ machine:
   java:
     version: openjdk8
   post:
-    # Manually install very recent version of cassandra; unless we create a user we have to run it as root
-    # uncomment below when datastax has version 3.9 available. at time of commit, latest was 3.7
-    #- curl -SL http://www-us.apache.org/dist/cassandra/3.9/apache-cassandra-3.9-bin.tar.gz | tar xz
-    - curl -SL http://cassci.datastax.com/job/trunk/lastSuccessfulBuild/artifact/build/apache-cassandra-3.10-SNAPSHOT-bin.tar.gz | tar xz
-    - sed -i -e 's/INFO/WARN/g' apache-cassandra-*/conf/logback.xml
-    - sudo ./apache-cassandra-*/bin/cassandra -R
+    # Manually install very recent version of cassandra
+    - curl -SL http://downloads.datastax.com/community/dsc-cassandra-3.0.8-bin.tar.gz | tar xz
+    - sed -i -e 's/INFO/WARN/g' dsc-cassandra-*/conf/logback.xml
+    - sudo ./dsc-cassandra-*/bin/cassandra
```

Comment:
Since you arent always on gitter or watching issues, you might not realize
that not everyone rolls their own or supports their own cassandra. I often
am in the middle of this. It was very hard to get people to even move to
cassandra 2.1. We cant whistle by this reality.

If the sasi aspect is strictly important to you, id suggest a
self-supported separate repo for 3.9 (even in zipkin org). Since I put in a
significant amount of effort I would like to see a 3.0 version of our work
here. Having that here will also easy your future work and also mean the
dependency graph will be compatible when completed